Composer/packages/adaptive-form/src/components/fields/TextField/TextField.base.tsx

```diff
@@ -0,0 +1,610 @@
+import debounce from 'lodash/debounce';
+
+const getClassNames = classNamesFunction<ITextFieldStyleProps, ITextFieldStyles>();
+
+/** @internal */
+export interface ITextFieldState {
+  /** Is true when the control has focus. */
+  isFocused?: boolean;
+
+  /**
+   * Dynamic error message returned by `onGetErrorMessage`.
+   * Use `this._errorMessage` to get the actual current error message.
+   */
+  errorMessage: string | JSX.Element;
+}
+
+/** @internal */
+export interface ITextFieldSnapshot {
+  /**
+   * If set, the text field is changing between single- and multi-line, so we'll need to reset
+   * selection/cursor after the change completes.
+   */
+  selection?: [number | null, number | null];
+}
+
+const DEFAULT_STATE_VALUE = '';
+const COMPONENT_NAME = 'TextField';
+
+export class TextFieldBase extends React.Component<ITextFieldProps, ITextFieldState, ITextFieldSnapshot>
+  implements ITextField {
+  public static defaultProps: ITextFieldProps = {
+    resizable: true,
+    deferredValidationTime: 200,
+    validateOnLoad: true,
+  };
+
+  /** Fallback ID if none is provided in props. Access proper value via `this.componentId`. */
+  private _fallbackId: string;
+  private _descriptionId: string;
+  private _labelId: string;
+  private _delayedValidate: (value: string | undefined) => void;
+  private _lastValidation: number;
+  private _latestValidateValue: string | undefined;
+  private _hasWarnedNullValue: boolean | undefined;
+  private _textElement = React.createRef<HTMLTextAreaElement | HTMLInputElement>();
+  private _async: Async;
+  /** Most recent value from a change or input event, to help avoid processing events twice */
+  private _lastChangeValue: string | undefined;
+
+  private _syncData;
+
+  public constructor(props: ITextFieldProps) {
+    super(props);
+
+    initializeComponentRef(this);
+    this._async = new Async(this);
+
+    if (process.env.NODE_ENV !== 'production') {
+      warnMutuallyExclusive(COMPONENT_NAME, props, {
+        errorMessage: 'onGetErrorMessage',
+      });
+    }
+
+    this._fallbackId = getId(COMPONENT_NAME);
+    this._descriptionId = getId(COMPONENT_NAME + 'Description');
+    this._labelId = getId(COMPONENT_NAME + 'Label');
+
+    this._warnControlledUsage();
+
+    let { defaultValue = DEFAULT_STATE_VALUE } = props;
+    if (typeof defaultValue === 'number') {
+      // This isn't allowed per the props, but happens anyway.
+      defaultValue = String(defaultValue);
+    }
+    this.state = {
+      isFocused: false,
+      errorMessage: '',
+    };
+
+    this._delayedValidate = this._async.debounce(this._validate, this.props.deferredValidationTime);
+    this._lastValidation = 0;
+
+    this._syncData = debounce((defaultValue: string | undefined) => {
+      if (this._textElement.current && defaultValue !== undefined && defaultValue !== this._textElement.current.value) {
+        this._textElement.current.value = defaultValue;
+      }
+    }, 300);
```

Comment:
This should probably come from a prop with a default value

Reply:
Hi, @LouisEugeneMSFT this is a quick fix for the issue,  we will remove this if we have a final solution. I want to keep the props the same as the FluentUI's TextField, then if we can use FluentUI again, we can replace it quickly. I have add some comments in this part of the code.